colors/embark.vim

```diff
@@ -325,6 +325,15 @@ call s:h("CtrlpMatch", {"fg": s:yellow})
 call s:h("NERDTreeDir", {"fg": s:blue})
 call s:h("NERDTreeFlags", {"fg": s:green})
 
+" nvim LSP
+call s:h ("LspDiagnosticsError", {"fg": s:red, "bg": s:deep_space})
```

Comment:
Would you be willing to change the `deep_space` reference to `bg_subtle` and `space` to `bg`?

I wanted the palette to have named colors to really breathe life into this theme (similar to what nord does). But I also don't want when people are contributing to be frustrated by the names so there is  some places where I map the names to more UI centered naming.

Reply:
Of course, updating right now.

Reply:
To be honest I started out using those names but got lazy because it was easier to use the colors I could see (via https://github.com/norcalli/nvim-colorizer.lua).

I agree with you that it's nicer to use those variables :)

<img width="1304" alt="Screenshot 2020-08-17 at 18 06 05" src="https://user-images.githubusercontent.com/145502/90417771-7e8b1280-e0b4-11ea-9042-b1943ca42046.png">

PR updated.

Reply:
Perfect! Thanks for amending the last commit too. I plan at some point putting up guidelines for contributing and one of them is going to be around a good git history!